### PR TITLE
Feature/#219 design improvement

### PR DESCRIPTION
**Pull request by jflute**
o use h1 tag as page title
o change h4 to h3 without semantic ui classes
o change h3 to h2 without semantic ui classes

### CODE REVIEW

File: src/static/app/index.css [hunk 1]
```diff
@@ -1,35 +1,93 @@
...
 =======================================================*/
+
+/* DBFlute-like background */
 body {
   background: transparent url(../image/parts/background.png) repeat-x;
 }
 
+h1 {
+	margin: 0 -20px 40px -20px;
+	padding: 40px 0 10px 140px;
```
Comment (EgumaYuto) [inline src/static/app/index.css:12]:
```suggestion
	margin: 0 0 40px 0;
	padding: 40px 20px 10px 140px;
```
avoid unnecessary negative margin.

File: src/static/app/client/client-content/ex-alter-check.tag [hunk 2]
```diff
@@ -11,16 +11,16 @@
...
       <alter-check-form ref="altercheckform" projectname="{ opts.projectName }" updatehandler="{ updateBegin }" />
     </section>
 
     <!-- Step 2 -->
     <section show="{ isEditing() }">
-      <h4 class="ui header">Step2. Execute AlterCheck</h4>
+      <h3>Step2. Execute AlterCheck</h3>
 
-      <h5 class="ui header">Open Editing Alter SQL Files</h5>
+      <h5>Open Editing Alter SQL Files</h5>
```
Comment (EgumaYuto) [inline src/static/app/client/client-content/ex-alter-check.tag:23]: Please use h4 after h3.

File: src/static/app/index.css [hunk 3]
```diff
@@ -1,35 +1,93 @@
...
   font-size: 150%;
-}
\ No newline at end of file
+}
+
+/* ======================================================
+  SchemaPolicyCHeck
+=======================================================*/
+
+h5.spolicy-category {
```
[the remark this answers is not in the document]
Comment (jflute) [reply src/static/app/index.css:89]: The "spolicy" means "SchemaPolicy" used in DBFlute world.

e.g. DfSPolicyChecker.java
https://github.com/dbflute/dbflute-core/blob/master/dbflute-engine/src/main/java/org/dbflute/logic/doc/spolicy/DfSPolicyChecker.java

File: src/static/app/index.css (hunk 1 above)
Comment (jflute) [reply src/static/app/index.css]: Thanks, fixed it

File: src/static/app/client/client-content/ex-alter-check.tag (hunk 2 above)
Comment (jflute) [reply src/static/app/client/client-content/ex-alter-check.tag]: fixed it like this:
https://github.com/dbflute/dbflute-intro/pull/231/files#diff-91510226fa32ca6037e514a4e542b333R98

File: src/static/app/index.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (EgumaYuto) [reply src/static/app/index.css:89]: Thanks! 🙇

**Pull request merged**